### PR TITLE
[WIP] Add Chain to monitor TOF Compressed data

### PR DESCRIPTION
**Pull request by njacazio**
- Add task, checker, json to monitor TOF compressed data as obtained from the O2 workflow `o2-tof-compressor`

### CODE REVIEW

File: Modules/TOF/CMakeLists.txt [hunk 1]
```diff
@@ -2,16 +2,29 @@
...
 
 target_include_directories(
   QcTOF
   PUBLIC $<INSTALL_INTERFACE:include>
          $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-  PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src)
-
-target_link_libraries(QcTOF PUBLIC QualityControl)
+  PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src
+         ${O2_ROOT}/include/TOFWorkflow
```
Comment (Barthelemy) [inline Modules/TOF/CMakeLists.txt]: This should not be needed if I am not mistaken. The imported targets defined later in `target_link_libraries` are enough.

File: Modules/TOF/CMakeLists.txt [hunk 2]
```diff
@@ -2,16 +2,29 @@
...
+         ${O2_ROOT}/include/TOFCompression
+         ${O2_ROOT}/include/DataFormatsTOF
+         ${O2_ROOT}/include/TOFBase )
+
+target_link_libraries(QcTOF PUBLIC QualityControl
+                                    O2::TOFBase
+                                    O2::TOFWorkflow
+                                    O2::DataFormatsTOF
+                                    O2::TOFCompression)
```
Comment (Barthelemy) [inline Modules/TOF/CMakeLists.txt]: It seems that `O2::TOFCompression` is missing. Is it something that you have not yet merged into O2 ?

Comment (njacazio) [reply Modules/TOF/CMakeLists.txt]: Ah yes, as you say I used a branch not merged in dev. I forgot about this. I will verify with Roberto when he will open his PR. Thank you Barth!

File: Modules/TOF/CMakeLists.txt (hunk 1 above)
Comment (njacazio) [reply Modules/TOF/CMakeLists.txt]: Dear Barthelemy, indeed, as you suggested I modified the CMakeLists file

**Comment by njacazio** [comment]
The dependency from O2 should be now merged. I guess we can continue the review process.

**Comment by njacazio** [comment]
Dear Barthelemy, I will update the PR to the latest changes in O2. 
Let's still wait a bit before merging this.

**Comment by Barthelemy** [comment]
Dear Nicolò, 
Just remove the `[WIP]` when you are ready for review. 
Cheers,

**Comment by njacazio** [comment]
This PR depends on https://github.com/AliceO2Group/AliceO2/pull/3038

**Pull request closed without merging**